src/main/java/org/opentestsystem/delivery/logging/EventLogger.java

```diff
@@ -7,31 +7,149 @@
 import org.slf4j.Marker;
 import org.slf4j.MarkerFactory;
 
+import java.util.HashMap;
 import java.util.Map;
 
 import static java.lang.String.format;
+import static java.util.Collections.unmodifiableMap;
 
 /**
  * Centralized event logger.
  */
 public class EventLogger {
-    static private Logger logger = LoggerFactory.getLogger (EventLogger.class);
-    static private Marker marker = MarkerFactory.getMarker("event-logger");
-    static private ObjectWriter writer = new ObjectMapper().writer();
-
-    /**
-     * Log events to centralized logging.
-     *
-     * These events are intended to be sent to a logstash server and searchable using elastic search.
-     *
-     * @param event name of the event occurring
-     * @param fields additional event data to send for use in searching
-     */
-    public static void log(String event, Map<String, String> fields) {
-        try {
-            logger.info(marker, format("EVENT:%s JSON:({\"event_data\": %s})", event, writer.writeValueAsString(fields)));
-        } catch (Exception e) {
-            logger.warn(marker, format("exception occurred while logging event: %s", event), e);
+
+  private static final Logger logger = LoggerFactory.getLogger(EventLogger.class);
+  private static final Marker marker = MarkerFactory.getMarker("event-logger");
+  private static final ObjectWriter writer = new ObjectMapper().writer();
```

Comment:
Its a good point.  We haven't serialized types from those packages yet.  So far the functions have been stateless.  We may have to refactor the class to inject the json writer.